_data/rules.yml

```diff
@@ -998,6 +998,66 @@ groups:
           - name: instaclustr/cassandra-exporter
             doc_url: https://github.com/instaclustr/cassandra-exporter
             rules:
+              - name: 'Cassandra Node is unavailable
+                description: 'Cassandra Node is unavailable - {{ $labels.cassandra_cluster }} {{ $labels.exported_endpoint }}'
+                query: 'sum(min_over_time(cassandra_endpoint_active{cassandra_cluster=~".*"}[1m])) by (cassandra_cluster,instance,exported_endpoint) < 1'
```

Comment:
Is it necessary to combine `min_over_time()[1m]` and `for: 2m`?

What about removing both? You will get the alert as soon as a node is unavailable.

Reply:
yes, they are gone